### PR TITLE
[5.4] Removes useless else statements

### PR DESCRIPTION
**Pull request by nunomaduro**

### CODE REVIEW

File: src/Illuminate/Support/HigherOrderCollectionProxy.php [hunk 1]
```diff
@@ -27,8 +27,8 @@ class HigherOrderCollectionProxy
      */
     public function __construct(Collection $collection, $method)
     {
-        $this->method = $method;
         $this->collection = $collection;
+        $this->method = $method;
```
Comment (GrahamCampbell) [inline src/Illuminate/Support/HigherOrderCollectionProxy.php]: please don't change this order

File: src/Illuminate/Support/HigherOrderCollectionProxy.php [hunk 2]
```diff
@@ -48,7 +48,7 @@ public function __get($key)
      * Proxy a method call onto the collection items.
      *
      * @param  string  $method
-     * @param  array  $parameters
+     * @param  array   $parameters
```
Comment (GrahamCampbell) [inline src/Illuminate/Support/HigherOrderCollectionProxy.php]: please don't change that whitespace

File: src/Illuminate/Mail/Mailable.php [hunk 3]
```diff
@@ -129,15 +129,9 @@ public function queue(Queue $queue)
...
-            return $queue->connection($connection)->pushOn(
-                $queueName, new SendQueuedMailable($this)
-            );
-        } else {
-            return $queue->connection($connection)->push(
-                new SendQueuedMailable($this)
-            );
-        }
+        return $queue->connection($connection)->pushOn(
+            $queueName ?: null, new SendQueuedMailable($this)
```
Comment (GrahamCampbell) [inline src/Illuminate/Mail/Mailable.php]: you don't actually need the `?: null` here, right?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Illuminate/Mail/Mailable.php (hunk 3 above)
Comment (nunomaduro) [inline src/Illuminate/Mail/Mailable.php]: Otherwise $queueName could be false or 0.

**Pull request merged**